templates/celery.conf

```diff
@@ -0,0 +1,18 @@
+[program:{{ project_name }}-celery-{{ item.name }}]
+command={{ source_dir }}/dotenv.sh {% if new_relic_license_key %}{{ venv_dir }}/bin/newrelic-admin run-program {% endif %}{{ venv_dir }}/bin/celery -A {{ project_name }} {{ item.command }} {{ item.flags }}
```

Comment:
This fails if new_relic_license_key is undefined. Let's check use_newrelic instead, here and in gunicorn.conf.